xieffect/communities/services/tasks_rst.py

```diff
@@ -0,0 +1,100 @@
+from __future__ import annotations
+
+from flask_fullstack import counter_parser
+from flask_restx import Resource
+from flask_restx.reqparse import RequestParser
+
+from common import ResourceController, EventController, User
+from vault import File
+from .tasks_db import Task, TaskEmbed
+from ..base import Community, ParticipantRole
+from ..utils import check_participant
+
+# Set Tasks behavior here
+FILES_LIMIT = 10
+TASKS_PER_PAGE = 48
+CURRENT_ROLE = ParticipantRole.OWNER
+
+controller = ResourceController(
+    "communities-tasks", path="/communities/<int:community_id>/"
+)
+
+
+def check_files(
+    controller_: ResourceController | EventController, files: list[int]
+) -> list[int]:
+    """
+    - Delete duplicates from a list with file ids.
+    - Check files limit.
+    - Check if file id is existed.
```

Comment:
- ... list ~~with~~ of ...
- Check ~~files~~ list length limit
- Check if all files exist

Reply:
\+